.travis.yml

```diff
@@ -23,9 +23,5 @@ after_script:
     # Don't release on PR builds
     $TRAVIS_PULL_REQUEST = "false"
   ]]; then
-    # Authenticate NPM
-    echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
-    # Set version to timestamp
-    npm --no-git-tag-version version $($(npm bin)/json -f package.json version)-prerelease.$(date +%s)
-    npm publish
+    npm run semantic-release
   fi
```

Comment:
Do we still need `travis-after-all` or does `semantic-release` somehow handle that internally?